.gitignore

```diff
@@ -4,6 +4,17 @@
 erl_crash.dump
 *.ez
 
+ui/node_modules
```

Comment:
Please remove specific **apps** stuff from the global .gitignore and add them to each specific app's .gitignore. For example, there is already a .gitignore inside _apps/webapp_